app/serializers/discourse_data_explorer/query_details_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ::DiscourseDataExplorer
+  class QueryDetailsSerializer < QuerySerializer
+    attributes :id,
```

Comment:
I think you can just add `:sql` here, all the other attributes should come from the base class, e.g.

https://github.com/discourse/discourse/blob/55316b9623cf772bec759e7392bc19c53e0de0e7/app/serializers/about_serializer.rb#L8-L10